src/components/common/DeleteCollectionModal/DeleteCollectionModal.tsx

```diff
@@ -49,6 +54,34 @@ export class DeleteCollectionModal extends React.Component<Props, State> {
     disableBodyScroll(this.modalTarget);
   };
 
+  getCollectionId = () => {
+    const collectionPageResult = matchPath(this.props.location.pathname, {
+      path: COLLECTION_URL,
+      exact: true,
+    });
+    return get(collectionPageResult, "params.collection_id");
+  };
+
+  submit = async () => {
+    const collectionId = parseInt(this.getCollectionId());
+    if (collectionId) {
+      const resp = await deleteCollection(this.context, collectionId);
+      console.log("delete response", resp);
+      let collections = await this.context.requestCollections({
+        limit: 100,
+        offset: 0,
+      });
+      const newCollections = collections.filter(
```

Comment:
Similar to previous comment. Does this actually do anything? Technically, if the `deleteCollection` mutation succeeded, then `this.context.requestCollections` should return the list with the just-deleted collection missing, and store the list into the context as well. Is that not the case?